.github/workflows/i18n_server_list_update.yaml

```diff
@@ -0,0 +1,37 @@
+name: i18n Updater
```

Comment:
This has the same name of an existing workflow, it should be changed 
https://github.com/mozilla-mobile/mozilla-vpn-client/actions/

Reply:
It also doesn't work because there is no Python setup step. Opening a PR.